src/styles/components/ec-btn/ec-btn.css

```diff
@@ -194,11 +194,11 @@
     }
 
     &.ec-btn--primary-reverse {
-      @apply tw-text-gray-8;
+      @apply tw-text-info-hover;
 
       &:hover {
-        @apply tw-bg-gray-8;
-        @apply tw-text-key-4;
+        @apply tw-text-gray-8;
+        @apply tw-bg-info-hover;
```

Comment:
I think you forgot to change --secondary-reverse. Now these buttons look like this:
![Screenshot from 2020-10-22 09-29-16](https://user-images.githubusercontent.com/5777410/96845950-2f14ea00-1449-11eb-8fec-ebafae6ff4c7.png)

Reply:
Yes, I forgot that one! Fix in the next commit